docs/guides/sdk/getting-started.md

```diff
@@ -77,6 +66,31 @@ const client = createDirectus('http://directus.example.com').with(rest());
 const client = createDirectus('http://directus.example.com').with(graphql());
 ```
 
+```ts [TypeScript]
+import { createDirectus } from '@directus/sdk';
+import { rest } from '@directus/sdk/rest';
+import { graphql } from '@directus/sdk/graphql';
+
+interface Article {
+	id: number;
+	title: string;
+	content: string;
+}
+
+interface Schema {
+	articles: Article[];
+}
+
+// Client with REST support
+const client = createDirectus<Schema>('http://directus.example.com').with(rest());
+
+// Client with GraphQL support
+const client = createDirectus<Schema>('http://directus.example.com').with(graphql());
```

Comment:
Very tiny potential tweak, but since we now have two `client` variables with the same name here, maybe we can do `restClient` and `graphqlClient` respectively instead.

Reply:
I think this one is fine - it's implied from the comments it's either/or